### DISCUSSION TITLE
When will the same functionality of GitHub Copilot be available in JetBrains IDEs?

### DISCUSSION THREAD

**Discussion by martenpoferl** [community/community#132216] [Copilot Conversations] [answered]
Hello, when comparing the GitHub Copilot (+ Chat) plugin in VSCode with the plugin in JetBrains IDEs, there is functionality missing in JetBrains IDEs, but the cost is the same for both. When will you add the full functionality to the JetBrains IDEs?

**Comment by ChandruMuthuswamy** [comment]
GitHub Copilot's functionality in JetBrains IDEs is currently not as fully featured as in VSCode. However, GitHub has been working on expanding the capabilities of Copilot for JetBrains products. As of now, GitHub Copilot Chat has been made available in a private beta for JetBrains IDEs like IntelliJ, PyCharm, WebStorm, and others【9†source】. 

This private beta includes features that allow for natural language assistance similar to what is available in VSCode. The private beta is accessible to Copilot Business customers and Copilot Individual users, who can sign up for access【10†source】.

For full functionality and future updates, keep an eye on the GitHub blog and changelogs. If you encounter issues, providing feedback can help improve the tool as it continues to develop.

**Reply by martenpoferl** [reply]
Thank you for you reply. The sources are not linked.

**Reply by ChandruMuthuswamy** [reply, marked as answer]
GitHub Copilot's functionality in JetBrains IDEs is currently not as fully featured as in VSCode. However, GitHub has been working on expanding the capabilities of Copilot for JetBrains products. As of now, GitHub Copilot Chat has been made available in a private beta for JetBrains IDEs like IntelliJ, PyCharm, WebStorm, and others​ ([The GitHub Blog](https://github.blog/changelog/2023-11-08-copilot-in-jetbrains-ides-chat-is-now-available-in-private-beta-new-welcome-guide/))​.

This private beta includes features that allow for natural language assistance similar to what is available in VSCode. The private beta is accessible to Copilot Business customers and Copilot Individual users, who can sign up for access​ ([GitHub](https://github.com/orgs/community/discussions/110614))​.

For full functionality and future updates, keep an eye on the GitHub blog and changelogs. If you encounter issues, providing feedback can help improve the tool as it continues to develop.

Is that sources are linked now? If it is helpful to you then mark it as correct answer! and make sure you follow me too. I will be helpful for the other questions as well. Thank you

**Comment by Nico04** [comment]
I'm looking for the "Modify using Copilot" feature in Android Studio, do you have any idea when it will be available ?

**Comment by nataliedeadey** [comment]
I am also looking for **'Copilot Edits'** and the **GPT options**, including 'GPT-40,' 'Claude 3.5 Sonnet (Preview),' 'o1-mini (Preview),' and the 'o1-preview' feature in GoLand. Please let us know the exact date and time when you will provide this to JetBrains + Copilot users.

**Comment by imyhxy** [comment]
Where to report the issue about the github copilot plugin in jetbrians ide?

**Reply by RubenGamarrarodriguez-tomtom** [reply]
from what I see on https://plugins.jetbrains.com/plugin/17718-github-copilot, this discussion site or https://support.github.com/ seems like the provided options
